dominion/frontend/.eslintrc.json

```diff
@@ -1,16 +1,27 @@
 {
   "root": true,
+  "parser": "babel-eslint",
   "extends": ["airbnb", "plugin:prettier/recommended", "prettier/react"],
   "env": {
     "browser": true,
     "jest": true
   },
   "rules": {
+    "react/prop-types": 0,
+    "react/forbid-prop-types": 0,
+    "import/no-named-as-default": 0,
```

Comment:
Don't disable rules, _fix_ the issues they raise. All these are fixable in code.